src/Schema/Column/ColumnFactoryInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Schema\Column;
+
+/**
+ * The interface must be implemented by a column factory class. It should create a column schema for a database column
+ * type and initialize column information.
+ *
+ * @psalm-import-type ColumnInfo from ColumnSchemaInterface
+ */
+interface ColumnFactoryInterface
```

Comment:
```suggestion
interface ColumnSchemaFactoryInterface
```

Reply:
The word `Schema` is planned to be removed from the names of classes associated with columns